modules/values-minimal.yaml

```diff
@@ -1,3 +1,4 @@
 # CE Bundle "Minimal"
 deckhouseConfigEnabled: true
 deckhouseEnabled: true
+externalModuleManagerEnabled: true
```

Comment:
Why we need this module in minimal ?

Reply:
Don't we?
Looks like some core feature - ability to add external modules to any Deckhouse installation

Reply:
Really, for this experimental module we can disable it for the minimal bundle